src/main/java/seedu/address/model/student/Assignment.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.student;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public abstract class Assignment {
```

Comment:
```suggestion
/**
 * Represents an assignment (that can be of type "homework" or "test")
 * <p></p>
 * <b>Note:</b> Abstract class that needs to be implemented
 *
 * @author mayuanxin1234
 * @since (add date here)
 */ 
```